sites/main-site/src/content/events/2025/hackathon-march-2025/memorial-sloan-kettering-cancer-center-new-york.mdx

```diff
@@ -26,25 +26,37 @@ Organizers:
 
 ## Venue
 
-Department of Computational Oncology of Memorial Sloan Kettering Cancer Center.
-Joy Building. 321 E 61st, New York, US. 10065
+    Memorial Sloan Kettering Cancer Center
+    Department of Computational Oncology
+    Macklowe Building
+    323 E 61st, New York, US, 10065
```

Comment:
I think the geoCoordinates above need to be updated then.

Reply:
Done. Thanks